src/lib/modules/Document/DocumentMetadataExtensions.js

```diff
@@ -5,19 +5,24 @@ import React from 'react';
 import { Divider, Table } from 'semantic-ui-react';
 
 export const DocumentMetadataExtensions = ({ extensions, showDivider }) => {
+  const documentExtensions = invenioConfig.DOCUMENTS.extensions;
   return (
     <React.Fragment>
-      {showDivider && (
-        <Divider horizontal>{invenioConfig.DOCUMENTS.extensions.label}</Divider>
-      )}
+      {showDivider && <Divider horizontal>{documentExtensions.label}</Divider>}
       <Table definition>
         <Table.Body>
           {_keys(extensions).map((key) => (
             <Table.Row key={key}>
               <Table.Cell width={4}>
-                {invenioConfig.DOCUMENTS.extensions.fields[key].label}
+                {documentExtensions.fields[key].label}
+              </Table.Cell>
+              <Table.Cell>
+                {documentExtensions.fields[key].type === 'boolean'
```

Comment:
since we do this, maybe it makes sense to have a global solution for the boolean values display, not only for the extensions ? ping @ntarocco

Reply:
Yes it makes sense, e.g. an utility `prettyPrintExtensionField(type, value)`